build-parsers.sh

```diff
@@ -28,5 +28,7 @@ fi
 
 #Generate the haxe and hxml parsers
 java -jar grammar-kit.jar gen ./grammar/haxe.bnf
-java -jar grammar-kit.jar gen ./src/com/intellij/plugins/haxe/hxml/hxml.bnf
+#java -jar grammar-kit.jar gen ./src/common/com/intellij/plugins/haxe/hxml/hxml.bnf
```

Comment:
Yes, these were local variable names.
If one can run `build-parsers.sh` to achieve consistent result regardless of the IDEA version & Grammar Kit plugin in it, then apparently we should go with it. So I vote for this script.